ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -238,7 +238,16 @@ function filter_test_by_capability() {
     done
     echo_e2e_tags
 }
+function filter_test_by_profile() {
+    local enabledgates
+    enabledgates="$(oc get featuregate cluster -o yaml | yq '.status.featureGates[].enabled[].name' | sort | uniq)"
+    if ! (grep --ignore-case --quiet 'VSphereMultiVCenters' <<< "$enabledgates") ; then
```

Comment:
```suggestion
    local platform, enabledgates
    platform="$(oc get infrastructure cluster -oyaml|yq '.status.platformStatus.type' |  tr '[:upper:]' '[:lower:]')"
    enabledgates="$(oc get featuregate cluster -o yaml | yq '.status.featureGates[].enabled[].name' | sort | uniq)"
    if [[ "$platform" == "vsphere" ]] && ! (grep --ignore-case --quiet 'VSphereMultiVCenters' <<< "$enabledgates") ; then
```
should solve the issue that other platforms clusters also have the VSphereMultiVCenters featuregate enabled.